### PR TITLE
v0.5.8

### PR DESCRIPTION
**Pull request by japaric**
required to make a release

we can publish now or wait for #120

### CODE REVIEW

**Comment by japaric** [comment]
added #120 to the changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,27 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+- `SCB` gained methods to set and get the priority of system handlers like
+  SVCall and SysTick.
+
+- `NVIC` gained *static* methods, `pend` and `unpend`, to set and clear the
+  pending state of interrupts.
+
+### Changed
+
+- The `NVIC.{clear,set}_pending` methods have been deprecated in favor of
+  `NVIC::{,un}pend`.
```
Comment (therealprof) [inline CHANGELOG.md]: `{,un}` reads like a typo to me.

Comment (japaric) [reply CHANGELOG.md]: changed this line to {unpend,pend}

**Comment by japaric** [comment]
Published: https://crates.io/crates/cortex-m/0.5.8

**Pull request merged**